### PR TITLE
Structure n tests

### PR DESCRIPTION
**Pull request by RyanHedges**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'sinatra/activerecord/rake'
+require './hedgeslist_app.rb'
+
+
+namespace :db do
+
+  desc "create the database"
+  task :createlist do
+    puts "creating database"
+  end
```
Comment (zspencer) [inline Rakefile:10]: What does this rake task do?

File: spec/feature_tests/guest_homepage_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  scenario "visit new page from home page to create post" do
+      visit "/"
+      click_on "post something"
+      expect(page).to have_content "Post all the things!"
+  end
+
+  scenario "visit all sales page from home page" do
+    visit "/"
+    click_on "browse our listings"
+    expect(page).to have_content "Listings of all the things!"
```
Comment (zspencer) [inline spec/feature_tests/guest_homepage_spec.rb:23]: I would recommend part of your `arrange` be creating a few listings and asserting they're included on this page.

That makes the test a bit more useful.

File: spec/feature_tests/guest_post_listing_spec.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+feature "guest can create listings" do
+  visit "/post-a-sale" do
+    fill_in 'listing_title', with: 'i want to sell this'
+    fill_in 'listing_description', with: 'This is the coolest thing. you should buy it.'
+  end
+  click_on 'post your listing'
```
Comment (zspencer) [inline spec/feature_tests/guest_post_listing_spec.rb:8]: Where is your assert here?

File: views/post_a_sale.erb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<h1>Post all the things!</h1>
+
+<h3>Give a title and description for the things that you are going to sell.</h3>
+
+<form method="post" action="/post-a-sale">
+    <input type="text" name="listing_title" placeholder="Listing Title">
+    <br>
+    <textarea name="listing_description" placeholder="Description please" spellcheck="true" rows="13" cols="75"></textarea>
```
Comment (zspencer) [inline views/post_a_sale.erb:8]: No submit button?

**Comment by RyanHedges** [comment]
Hey! 👋 

I'm closing this PR. I'm now using [Pullwalla](https://pullwalla.com/) to see statuses of all my pull requests across all projects. This is stale and creating noise. Thanks for the memories! 🤟

**Pull request closed without merging**